content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -245,6 +241,12 @@ different Kubernetes components.
 | `EnableAggregatedDiscoveryTimeout` | `true` | Deprecated | 1.16 | - |
 | `EnableEquivalenceClassCache` | `false` | Alpha | 1.8 | 1.14 |
 | `EnableEquivalenceClassCache` | - | Deprecated | 1.15 | - |
+| `EndpointSlice` | `false` | Alpha | 1.16 | 1.16 |
+| `EndpointSlice` | `false` | Beta | 1.17 | 1.17 |
+| `EndpointSlice` | `true` | Beta | 1.18 | -  |
+| `EndpointSlice` | `true` | Stable | 1.21 | -  |
+| `EndpointSliceNodeName` | `false` | Alpha | 1.20 | |
+| `EndpointSliceNodeName` | `true` | Stable | 1.21 | |
```

Comment:
Need to update WindowsEndpointSliceProxying feature gate beta here as well.

Reply:
Thanks for catching that!